American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,48 @@
+|   |   |
+|---|---|
+| Title               |  Questioning the Future: American Elections and Crypto from a Cypherpunk Perspective |
+| Author              | João Paulo Sant'Anna da Silva |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+
+Thesis statement - What are the implications of the American election to crypto? will they change something?
+
+Outline: 
+
+Introduction:
+
+Emphasizing the importance of the 2024 American elections and their potential ramifications for the global cryptocurrency landscape.
+
+Elections and Government Intervention:
+
+Questioning government influence: Analyzing the potential impact of the elections on government policies towards cryptocurrency and the cypherpunk ethos.
+Cypherpunk skepticism: Examining the cypherpunk perspective on government intervention in the crypto space and its implications for privacy and decentralization.
+
+Republican vs. Democratic Policies:
+
+Republican stance: Investigating potential Republican policies regarding cryptocurrency 
+Democratic perspective: Evaluating Democratic approaches to cryptocurrency 
+
```

Comment:
I'm quite sceptical here. I think before we make any claims about these party policies we should have hard evidence. I'd like to see links to research that puts in it black and white what they are aiming to achieve in terms of crypto.

Before you start writing this piece I think it would also be best to provide links to these policies. It would also be great to add quotes from these documents, here in the comments, that shows us what you'll be scrutinising. 

I think if we're going to write a piece like this we need to scrutinise official documentation not just word of mouth.